packages/website/src/templates/Member.vue

```diff
@@ -1,9 +1,68 @@
 <template>
-  <div />
+  <page-background>
+    <navbar-padding />
+    <b-container class="member-padding">
+      <member-profile :profile="profile" />
+    </b-container>
+  </page-background>
 </template>
 
+<style lang="scss" scoped>
+.member-padding {
+  margin: 2rem;
+}
+</style>
+
+<page-query>
+query DTIMember($path: String!) {
+  member: member(path: $path) {
+    netid
+    image       
```

Comment:
How can this field still work after #294

Reply:
The site still uses the GraphQL data layer for members and projects, as they are collections and not individual pages (which I stripped out)

In #294 I just switched the collection from using the generated JSON to using the raw JSON.

Reply:
#294 removed the image field, so should image field also be removed here?

Reply:
Ohh, member has an explicit gql schema because implicit type inference broke when members had differing fields/field types. Additionally, the Member type definition still includes it.

I pushed a commit to full strip `image` out.